recipes/curlpp/all/conanfile.py

```diff
@@ -50,26 +40,23 @@ def requirements(self):
         self.requires("libcurl/8.9.1", transitive_headers=True, transitive_libs=True)
 
     def validate(self):
-        if self.settings.compiler.get_safe("cppstd"):
-            check_min_cppstd(self, self._min_cppstd)
-        else:
-            if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "6":
-                raise ConanInvalidConfiguration("${self.ref} requires C++11. Please set 'compiler.cppstd=11'.")
-            if self.settings.compiler == "apple-clang" and Version(self.settings.compiler.version) < "14":
-                raise ConanInvalidConfiguration("${self.ref} requires C++11. Please set 'compiler.cppstd=11'.")
+        check_min_cppstd(self, 11)
+        if self.settings.compiler == "gcc" and Version(self.settings.compiler.version) < "6":
+            raise ConanInvalidConfiguration("${self.ref} requires C++11. Please set 'compiler.cppstd=11'.")
```

Comment:
the message in this error does not match the condition - this will raise on gcc <6 regardless of `compiler.cppstd`